src/commands/record.ts

```diff
@@ -0,0 +1,22 @@
+import robot from "robotjs";
+import { CursorLocation } from "../types/CursorLocation";
+import recordings from "../persistence/recordings";
+import { verbose } from "../utils/logger";
+
+export function record({ interval }: { interval: number }) {
+  verbose("Recording mouse movement.");
+  verbose(`Polling duration set to ${interval} milliseconds`);
+  verbose("Press CTRL+C to finish.");
```

Comment:
Might wanna make only the middle dude or the first two verbose logs, the last one might be relevant instruction for normal usage